client.js

```diff
@@ -3,8 +3,10 @@ const logger = require('winston');
 const settings = require('./settings.json');
 const { helpMsg, welcomeMsg } = require('./messages')
 const db = require('./db.js');
-const { fetchAndFormatAuctionData } = require("./wikiHandler");
-
+const { fetchAndFormatAuctionData, fetchImageUrl, fetchWikiPricing } = require("./wikiHandler");
+const { SERVER_COLOR } = require('./wikiHandler')
```

Comment:
Why is this a second line of the same import? Is it a style thing? Any performance repercussions for doing it this way? I don't know how it works in JS, in Python this WOULD be totally normal/fine to do, but in some other languages a double import is not good. No clue.

Reply:
I didn't notice, good catch.  Shouldn't error in JS but no reason it shouldn't be one import.